prog/vm/github_runner.rb

```diff
@@ -131,10 +131,21 @@ def before_run
   end
 
   label def wait_concurrency_limit
-    Clog.emit("Waiting for customer concurrency limit") { {github_runner: github_runner.values} }
-
     hop_allocate_vm if concurrency_available?
-    nap 5
+
+    # check utilization, if it's high, wait for it to go down
+    utilization = VmHost.where(location: "github-runners", allocation_state: "accepting", arch: github_runner.label.include?("arm") ? "arm64" : "x64").select_map {
+      sum(:used_cores) * 100.0 / sum(:total_cores)
+    }.first.to_f
+
+    unless utilization < 60
```

Comment:
nit: it's harder to digest
```suggestion
    if utilization > 60
```